typescript.js

```diff
@@ -15,7 +15,9 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-parameter-properties': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off'
-
+    
+    "@typescript-eslint/prefer-includes': 'on'
```

Comment:
```suggestion
    "@typescript-eslint/prefer-includes': 'error'
```

I think this is the correct way to turn it on>